src/setup/rgw_setup

```diff
@@ -146,36 +175,37 @@ class UpgradeCmd(SetupCmdBase):
     def __init__(self, args: dict):
         """Initialize super class members."""
         super().__init__(args)
-        self.config_path = args.config
+        self._cortx_gconf = self.get_cortx_config(args.config)
 
     def process(self):
+        self.initialize_logging(self._cortx_gconf)
         Rgw.validate('upgrade')
-        rc = Rgw.upgrade(self.config_path)
+        rc = Rgw.upgrade(self._cortx_gconf)
```

Comment:
Minimise use of cortx. Replace "_cortx_gconf" with "conf" everywhere.

Reply:
done